static/js/components/LearningResourceCard_test.js

```diff
@@ -242,11 +225,70 @@ describe("LearningResourceCard", () => {
       )
       assert.equal(wrapper.find(".drag-handle").exists(), reordering)
       assert.equal(wrapper.find("Subtitle").exists(), !reordering)
-      assert.equal(
-        wrapper.find(".availability-price-favorite").exists(),
-        !reordering
-      )
+      assert.equal(wrapper.find(".start-date-favorite").exists(), !reordering)
       assert.equal(wrapper.find(".cover-image").exists(), !reordering)
     })
   })
+
+  //
```

Comment:
nit: you don't need the `//` here, it's only necessary before a line that starts with `;`, like

```js
;[true, false].forEach(...)
```

because the code formatter sees `;` and tries to delete any blank lines before it